.github/.env

```diff
@@ -1,5 +1,7 @@
 CARGO_TERM_COLOR=always
 NODE_VERSION=16.x
 PROGRAMS=["system-extras", "token-extras"]
-RUST_VERSION=1.65.0
-SOLANA_VERSION=1.14.13
+RUST_VERSION=1.70.0
+SOLANA_VERSION=1.17.20
```

Comment:
So is it time for us to start making all our CI use 1.17.20 probably?

Reply:
Probably, yeah